boudicca.base/publisher-event-html/src/main/kotlin/base/boudicca/publisher/event/html/controller/StartPageController.kt

```diff
@@ -14,7 +14,7 @@ import org.springframework.web.servlet.ModelAndView
 @RequestMapping("/")
 class StartPageController @Autowired constructor(private val eventService: EventService) {
 
-    private val PAGE_TITLE = "Boudicca.Events - Open source event collector and publisher software"
+    private val PAGE_TITLE = "Boudicca.Event - find accessible events in Austria"
```

Comment:
the page is still called "Boudicca.Events", but the additional text looks fine for me

Reply:
done